### PR TITLE
OHRM5X-875: Email Configuration password mask added

### PR DESCRIPTION
**Pull request by VageeshOrangeHRM**
This PR Contains following fixes:
1. OHRM5X-875 - Password masking in SMTP password field

### CODE REVIEW

File: symfony/plugins/orangehrmAdminPlugin/Api/EmailConfigurationAPI.php [hunk 1]
```diff
@@ -291,13 +290,17 @@ public function setEmailConfiguration(EmailConfiguration $emailConfiguration)
...
-            $this->getRequestParams()->getStringOrNull(
-                RequestParams::PARAM_TYPE_BODY,
-                self::PARAMETER_SMTP_PASSWORD
-
-            )
+        $password = $this->getRequestParams()->getStringOrNull(
+            RequestParams::PARAM_TYPE_BODY,
+            self::PARAMETER_SMTP_PASSWORD
         );
+        if ($password !== '********') {
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmAdminPlugin/Api/EmailConfigurationAPI.php:297]: Please check when user input password as `********`, what should be the behaviour.
And I prefer to send this password as `null` from FE, if user not changing the password

File: symfony/client/src/orangehrmAdminPlugin/pages/emailConfiguration/ViewEmailConfiguration.vue [hunk 2]
```diff
@@ -136,6 +136,7 @@
                 v-model="emailConfiguration.smtpPassword"
                 :rules="rules.smtpPassword"
                 type="password"
+                onfocus="this.value=''"
```
Comment (RajithaKumara) [inline symfony/client/src/orangehrmAdminPlugin/pages/emailConfiguration/ViewEmailConfiguration.vue:139]:
```suggestion
                onfocus="smtpPassword=''"
```
??

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: symfony/plugins/orangehrmAdminPlugin/Api/EmailConfigurationAPI.php (hunk 1 above)
Comment (VageeshOrangeHRM) [reply symfony/plugins/orangehrmAdminPlugin/Api/EmailConfigurationAPI.php]: done

File: symfony/client/src/orangehrmAdminPlugin/pages/emailConfiguration/ViewEmailConfiguration.vue (hunk 2 above)
Comment (VageeshOrangeHRM) [reply symfony/client/src/orangehrmAdminPlugin/pages/emailConfiguration/ViewEmailConfiguration.vue]: done

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**